### PR TITLE
Upload files from previous project

### PR DESCRIPTION
**Pull request by prius2055**
In this branch, I have copied my todo list app files from the previous project.

The app contains the following features:

- Adding todos
- Deleting todos
- editing todos
- Marking all completed todos
- Clearing all completed todos

### CODE REVIEW

**Review by zunairkhan811 — CHANGES_REQUESTED** [review]
## Hi @prius2055  🙋‍♂️
Good job working on your first to do list project. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

Please consider the inline comment.

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
+    />
+    <title>To do List</title>
+  <script defer src="main.js"></script></head>
+  <body>
```
Comment (zunairkhan811) [inline dist/index.html:13]: It would be good if you kindly use the `main` tag in your HTML which will be good for your accessibility. 😉

Comment (zunairkhan811) [reply dist/index.html:13]: Linter checks are very important for removing errors please kinldy also consider linter checks as they are not passing on your repo. ❤️

**Review by zunairkhan811 — CHANGES_REQUESTED** [review]

File: dist/index.html (hunk 1 above)
Comment (prius2055) [reply dist/index.html:13]: Thanks for the feedback. I will take note of this and fix them accordingly.

**Pull request merged**